analyses.qmd

```diff
@@ -942,17 +962,14 @@ qqnorm(resid(q1_a2))
 qqline(resid(q1_a2))
 ```
 
-```{r}
-
-```
 
 Then by species: the 4 main spruce, rowan, birch and pine
 
 ```{r}
-modelData2 <- modelData %>%
+modelData2_SA <- modelData2 %>%
```

Comment:
@gunnaraustrheim , here you recreated (overskrev) `modelData2`. I will instead give it a new name `modelData2_SA `and base it on `modelData2`, not `modelData`. Now the `Taxa `and `HeightClass `field are present.